java/registry/src/main/java/dev/sunbirdrc/registry/helper/EntityStateHelper.java

```diff
@@ -70,6 +66,21 @@ void applyWorkflowTransitions(JsonNode existing, JsonNode updated, List<Attestat
         addAttestationStateTransitions(existing, entityName, modified, allContexts, attestationPolicies);
         addOwnershipStateTransitions(existing, entityName, updated, allContexts);
         ruleEngineService.doTransition(allContexts);
+        updated = removePasswordFields(entityName, updated);
```

Comment:
you can avoid returning this since updated is already passed in function as argument

Reply:
This is required since it is returning a new copy instead of modifying the updated